share/gpodder/extensions/youtube-dl.py

```diff
@@ -270,6 +275,10 @@ def __init__(self, gpodder_config=None):
         if not sys.stdout:
             logger.debug('no stdout, setting YoutubeDL logger')
             self._ydl_opts['logger'] = logger
+        if ytdl_config.geo_bypass:
+            self._ydl_opts['geo_bypass'] = True
+        if ytdl_config.geo_bypass_country != '':
```

Comment:
```suggestion
        if ytdl_config.geo_bypass_country:
```

Reply:
I haven't fixed this yet because I'm not sure if this patch is the right way to handle it, or if it even works without the third setting. A per-podcast setting would be more useful than a global setting that might cause issues for other podcasts.